frontend/packages/dev-console/integration-tests/support/pages/add-flow/catalog-page.ts

```diff
@@ -97,3 +98,21 @@ export const catalogPage = {
       .should('contain.text', partialCardName);
   },
 };
+
+export const catalogInstallPageObj = {
+  installHelmChart: {
+    install: '[data-test-id="submit-button"]',
+    chartVersion: '#form-dropdown-chartVersion-field',
+    yamlView: '#form-radiobutton-editorType-yaml-field',
+  },
```

Comment:
Please move this to pageObjects/add-flow-po.ts

Reply:
Sure, Done, Thanks